unlock-protocol-com/blog/sept-22-update.md

```diff
@@ -0,0 +1,31 @@
+---
+title: Unlock Protocol Product Updates for September 2022
+subTitle: New dashboard preview, support for Arbitrum and Avalanche, and improvements to the checkout experience
+authorName: Christopher Carfi
+publishDate: September 30, 2022
+description: Release Notes for September 2022. New Unlock Protocol dashboard preview, support for Arbitrum and Avalanche, and improvements to the checkout experience.
+image: /images/blog/sept-22-update/sept-22-banner.png
+---
+
+![sept-22-banner.png](/images/blog/sept-22-update/sept-22-banner.png)
+
+## Partners and integrations
+- Launched on [Arbitrum](https://unlock-protocol.com/blog/arbitrum)
+- Launched on [Avalanche](https://unlock-protocol.com/blog/avalanche)
+
+## Checkout 
+- [Support for fiat recurring subscriptions](https://unlock-protocol.com/blog/unlock-fiat-subscription)
+- [Support for Apple Pay and Google Pay mobile payments](https://unlock-protocol.com/blog/support-for-google-and-apple-pay)
+- [Password-protected NFT purchases](https://unlock-protocol.com/blog/password-required-hook) 
+
+## Members page
+- [Enhanced Members Page filters](https://unlock-protocol.com/blog/new-filters-members-page) 
```

Comment:
```suggestion
```

Reply:
We can remove this one, is part of an old Update, also the new dashboard includes the new members page

Reply:
thanks @kalidiagne — going to leave it in the list as it WAS one of the many things we shipped in september. we move so fast, some things change like that!